### ISSUE TITLE
Stealthed planets considered a target in round 1

### ISSUE THREAD

**Issue by MichaelCourtney** [freeorion/freeorion#4149]
2022-08-30.000f16b

A stealthed planet can not be seen if its stealth is higher then the enemies detection. However if combat is triggered by other ships in the system it is an eligible target on round 1. This is inconsistent with how stealth ships work.

**Comment by geoffthemedio** [comment]
planets are not ships... the fluff descriptions generally relate to hiding the stuff on the planet rather than the location of the planet entirely. how this should work is up for debate and adjustment, though.

**Comment by MichaelCourtney** [comment]
ok opened a thread for it:

https://www.freeorion.org/forum/viewtopic.php?f=6&t=12589

**Comment by Oberlus** [comment]
> planets are not ships... the fluff descriptions generally relate to hiding the stuff on the planet rather than the location of the planet entirely. how this should work is up for debate and adjustment, though.

Then the problem is defense and ofense structures of the planet should remain hidden until they attack, the enemies should not be able to target them.

In any case, those fluff-descriptions are nonsensical when it comes to colonization: so I see the planet but I cannot colonize it because I can't find the right spot to land? No way, the fluff here is "you know there is a planet somewhere in this system but you can't locate it".

**Comment by geoffthemedio** [comment]
> the fluff here is "you know there is a planet somewhere in this system but you can't locate it".

There are currently [several planet stealth-related specials](https://github.com/freeorion/freeorion/tree/master/default/scripting/specials/planet/monster_stealth) like "cloud cover" and "volcanic ash" that suggest otherwise.

**Comment by Oberlus** [comment]
> > the fluff here is "you know there is a planet somewhere in this system but you can't locate it".
> 
> There are currently [several planet stealth-related specials](https://github.com/freeorion/freeorion/tree/master/default/scripting/specials/planet/monster_stealth) like "cloud cover" and "volcanic ash" that suggest otherwise.

Yes, it makes no sense at all: because of some dense cloud or dust cover, you can't land on the planet with a ship?
It makes sense that the cloud cover doesn't let you know what's under it, open space to land? lots of weapons aimed at your head? But then you just get under the clouds and discover. No need for high detection technologies to land (or crash) on a planet.

But that is off-topic, sorry.

The point is: if you can't shoot a ship (mobile weapon in the space) until it reveals itself by attacking you, then you can't shoot a planetary defense (immobile or mobile weapon at the planet surface, under all those clouds and ashes) until it reveals itself...
